File: DesignPatternsExercises/java/BridgeDesignPattern/Solution/Theme.java
// public __________(8)__________ class Theme {
public abstract class Theme {
    private String backgroundColor;
    private String fontColor;

    public Theme(String backgroundColor, String fontColor) {
        this.backgroundColor = backgroundColor;
        this.fontColor = fontColor;
    }

    public String getBackgroundColor() {
        return backgroundColor;
    }

    public String getFontColor() {
        return fontColor;
    }
}

File: DesignPatternsExercises/ruby/bridge_design_pattern/solution/theme.rb
class Theme
  # attr_accessor __________(7)__________, :font_color
  attr_accessor :background_color, :font_color

  # def __________(8)__________(background_color, font_color)
  def initialize(background_color, font_color)
    @background_color = background_color
    @font_color = font_color
  end
end
